webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -1013,28 +1011,42 @@ <h3>Download Raw Crash Data and Minidumps</h3>
                   See <a href="{{ url('documentation:protected_data_access') }}">Protected data policy</a> for details.
                 {% endif %}
               </p>
-              <ul>
-                {% for name, url in raw_dump_urls %}
+              <ul class="list-inside list-disc">
+                {% for name, url in raw_data_urls %}
```

Comment:
I don't know what "raw" means anymore.

I moved generating the urls into the view code, so all this does is spit out what it knows about.